llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -842,19 +842,12 @@ getStrideFromAddRec(const SCEVAddRecExpr *AR, const Loop *Lp, Type *AccessTy,
   TypeSize AllocSize = DL.getTypeAllocSize(AccessTy);
   int64_t Size = AllocSize.getFixedValue();
 
-  // Huge step value - give up.
-  if (APStepVal->getBitWidth() > 64)
```

Comment:
This condition being true doesn't seem to happen in practice (and not covered by existing tests), but could the new could change behavior, e.g. if we have an AddRec with type i128 and a constant step of 1, for which `trySExtValue` would succeed, but the original check fail?